webpack.common.js

```diff
@@ -0,0 +1,18 @@
+const path = require('path');
+
+module.exports = {
+  entry: './src/index.js',
+  devtool: 'source-map',
+  output: {
+    path: path.resolve(__dirname, 'dist'),
+    library: 'ESNApiClient',
+    libraryTarget: 'umd'
```

Comment:
And we can also output `esm` here together with `umd`.

Reply:
Can you elaborate? Are you mentioning to https://github.com/purtuga/esm-webpack-plugin?

Reply:
Webpack needs a whole plugin to do that? :fearful:

I think we can create an issue, something like "Also output ESM", and handle it later. I think we'll need to use `rollup` to handle that.

Reply:
`rollup` is just much much simpler for building libraries. In fact it is made to build JS libraries.